runbotics/runbotics-orchestrator-ui/src/src-landing/components/BlogHeadImage/BlogHeadImage.tsx

```diff
@@ -9,7 +9,7 @@ import styles from './BlogHeadImage.module.scss';
 
 const BlogHeadImage: FC = () => (
     <div className={styles.image}>
-        <Image src={Background} alt='banner' className={styles.background} fill/>
+        <Image src={Background} alt='banner' className={styles.background}   style={{objectFit:'cover', margin:'auto'}} fill/>
```

Comment:
Next Image can't be styled using className.
https://nextjs.org/docs/pages/api-reference/components/image#style

Reply:
![image](https://github.com/runbotics/runbotics/assets/44121335/5bdaddb1-c20b-4672-a493-3f1a24197b3f)